src/sql/workbench/parts/commandLine/electron-browser/commandLine.ts

```diff
@@ -26,8 +27,24 @@ import { INotificationService } from 'vs/platform/notification/common/notificati
 import { IWorkbenchContribution } from 'vs/workbench/common/contributions';
 import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
 import { openNewQuery } from 'sql/workbench/parts/query/browser/queryActions';
+import { IURLService, IURLHandler } from 'vs/platform/url/common/url';
+import { parse } from 'vs/base/common/marshalling';
+import { getErrorMessage } from 'vs/base/common/errors';
+import { IDialogService } from 'vs/platform/dialogs/common/dialogs';
 
-export class CommandLineWorkbenchContribution implements IWorkbenchContribution {
+const connectAuthority = 'connect';
+
+interface SqlArgs {
+	_?: string[];
+	aad?: boolean;
+	database?: string;
+	integrated?: boolean;
+	server?: string;
+	user?: string;
+	command?: string;
+	provider?: string;
```

Comment:
should this be connectionProvider to disambiguate, should we ever add it to ParsedArgs?

Reply:
Yeah I can rename - trying to keep it short is the only thing. Which do you think is better (short vs. very clear)?